test/parallel/test-crypto-encoding-validation-error.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+// This test checks if error is thrown in case of wrong encoding provided into cipher.
+
+const assert = require('assert');
+const { createCipheriv, randomBytes } = require('crypto');
+
+const createCipher = () => {
+  return createCipheriv('aes-256-cbc', randomBytes(32), randomBytes(16));
+};
+
+{
+  const cipher = createCipher();
+
+  assert.throws(
+    () => cipher.update('test', 'bad1', 'hex'),
+    { message: /^Unknown encoding bad1$/, code: 'ERR_CRYPTO_UNKNOWN_ENCODING' }
+  );
+}
```

Comment:
This particular issue is in place also for Sign, Verify, Hash, Hmac.

I think fixing it is not necessary or at the very least not in scope for a non major change type fix.

Please replace this block with another test that ensures `Cannot change encoding` is actually still asserted.

```suggestion
{
  const cipher = createCipher();
  cipher.update('test', 'utf-8', 'utf-8')

  assert.throws(
    () => cipher.update('666f6f', 'hex', 'hex'),
    { message: /Cannot change encoding/ }
  );
}
```

Reply:
I am sorry, had a long vacation, looks good to me, I see you have applied them.